diff --git a/dnssd_osx.gemspec b/dnssd_osx.gemspec
index d9b715b..ed259bc 100644
--- a/dnssd_osx.gemspec
+++ b/dnssd_osx.gemspec
@@ -4,8 +4,7 @@ Gem.manage_gems
 
 spec = Gem::Specification.new do |s|
   s.name = 'dnssd'
-  s.version = "0.5.0"
-  s.platform = Gem::Platform::CURRENT
+  s.version = "0.6.0"
   s.summary = "DNS Service Discovery (aka Rendezvous) API for Ruby"
   s.files = Dir.glob("{lib,ext}/**/*").delete_if {|item| item.include?("CVS")}
   s.require_paths = ['lib', 'ext']
diff --git a/ext/extconf.rb b/ext/extconf.rb
index 6f1863e..838db56 100644
--- a/ext/extconf.rb
+++ b/ext/extconf.rb
@@ -27,6 +27,7 @@ end
 
 unless RUBY_PLATFORM.include? "darwin"
   have_library( "mdns", "DNSServiceRefSockFD" ) or
+  have_library( "dns_sd", "DNSServiceRefSockFD" ) or
     abort( "can't find rendezvous library" )
 end
 
